<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>349</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Your message:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="MessageLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="sendCommonMessageButton">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Send to all</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendPrivateMessageButton">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Send private</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="conversationButton">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Open a conversation</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTextBrowser" name="commonChatBrowser"/>
      <widget class="QTextBrowser" name="privateChatBrowser"/>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="forAdminButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>For admin</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain_menu">
    <property name="title">
     <string>Main menu</string>
    </property>
    <addaction name="actionOpen_another_client"/>
    <addaction name="separator"/>
    <addaction name="actionClose_this_client"/>
   </widget>
   <addaction name="menuMain_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_another_client">
   <property name="text">
    <string>Open another client</string>
   </property>
  </action>
  <action name="actionClose_this_client">
   <property name="text">
    <string>Close this client</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
